nikolay.braslavets/hw01/2.rb

```diff
@@ -0,0 +1,4 @@
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234',
+         yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+
+p hash.count { |k,_| k =~ /^yes/i }
```

Comment:
```suggestion
p hash.count { |k, _| k =~ /^yes/i }

```

Reply:
Git says that there is no new line in the end of the file.
This message is displayed because otherwise there is no way to tell the difference between a file where there is a newline at the end and one where is not. Diff has to output a newline anyway, or the result would be harder to read or process automatically.

Note that it is a good style to always put the newline as a last character if it is allowed by the file format. Furthermore, for example, for C and C++ header files it is required by the language standard.